clients/database/migrations/20230511134345_add_proxied_request_metrics_cachehit_column.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE IF EXISTS proxied_request_metrics ADD cache_hit boolean not null default false;
```

Comment:
track cache metrics in a separate table

Reply:
(or in cloudwatch / promethesus using an interface to abstract the backend from the service)